client/my-sites/site-settings/redirect-to/redirect-to.jsx

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
If we rename this file to `index.jsx`, we can import it with less code - `from 'my-sites/site-settings/redirect-to` instead of `from 'my-sites/site-settings/redirect-to/redirect-to`.

After we address this, let's make sure to address all affected imports (including the one in the README) accordingly.

Reply:
yes :)